website/docs/components/stepper/nav/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,87 @@
+## Usage
+
+### When to use
+
+- In complex flows that are broken up into multiple steps to more easily segment content.
+- To communicate what has and has not been completed in a multi-step flow.
+- For a single, on-page experience that requires completion.
+
+### When not to use
+
+- For non-complex flows, use the [form pattern](/patterns/form-patterns) on a single page.
+- For simple forms or when progress doesn't need to be tracked, use structured page content.
+- For progression that doesn’t require a user to remain on the page for, use the [Stepper List](/components/stepper/list).
+
+## Interactive vs non-interactive
+
+The Stepper Nav supports both interactive (default) and non-interactive variants.
+
+### Interactive steps
+
+A user is able to navigate backwards within the Stepper Nav, however, they cannot navigate forward unless they have completed the current step. This creates consistency for step flow behavior and order of operations.
+
+![](/assets/components/stepper/nav/stepper-nav-interactive-steps.png)
+
+![The user's mouse is hovering over step one of four, showing its hover state.](/assets/components/stepper/nav/stepper-nav-interactive-backwards-movement.png)
+
+### Non-interactive steps
+
+The non-interactive variant is for read-only purposes and does not allow backwards navigation.
+
+![](/assets/components/stepper/nav/stepper-nav-non-interactive-steps.png)
+
+## Content
+
+The Stepper Nav supports a title and an optional description. Content should be limited to only what is necessary, and long form content should live within the body of the page.
+
+### Title
+
+Titles provide an overview to help users understand what tasks need to be completed. Titles are required and should be brief.
+
+### Description
+
+Descriptions provide additional context about the step and are optional. We recommend that all steps either have a description or do not.
+
+!!! Do
+
+Implement consistent description visibility.
```

Comment:
```suggestion
Use descriptions consistently.
```

Reply:
"visibility" doesn't seem like the right word here as, to me, it implies that there may be invisible or hidden descriptions.